docs/requirements.txt

```diff
@@ -8,6 +8,9 @@ numpy
 ipywidgets
 matplotlib
 ase==3.22.1
+rdkit==2023.9.5
+stk==2024.9.23.1
+stko==2024.8.29.1
```

Comment:
Do we need to pin to specific versions here? The ASE pin is (supposed to be) temporary until we add a fix around an API change in ASE.

Reply:
These pins are also temporary because of an rdkit version issue that should change soon, I added a comment about this for future developers.

Reply:
perfect!